_projects/area_indicators.md

```diff
@@ -0,0 +1,66 @@
+---
+published: false # change to true once ready to post
+
+title: "Climate Change API Area Indicators" # project title inside quotes
+excerpt: "A high performance Scala API for calculating climate indicators over geospatial areas" # shows on project list page
+
+# project attributes
+requirements: # bullet list of requirements – one requirement per line – follow below format
+  - "Familiarity building or working with APIs"
+  - "Interest and/or exerience in functional programming required (Haskell, Scala, ML, etc.)"
+  - "Scala programming experience not required but preferred"
+  - "Python programming experience not required but preferred"
+tags: # one tag per line – spaces are allowed in tags – follow below format
+  - "scala"
+  - "climate change"
+difficulty: "Medium" # easy, medium, hard – pick one
```

Comment:
Based on the breadth of this and the skill a fellow would require to complete this, I think this would deserve a 'Hard' marking.

Reply:
:+1: done in https://github.com/azavea/fellowship-program-website/pull/66/commits/03b3425a3a49c653a2ad16190ed9c0c2d3729bfc